global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "7.0.100-rc.1.22431.12",
+    "dotnet": "7.0.100",
```

Comment:
It won't let me add a comment/suggestion since those lines weren't changed, but we should also update line 12 of this file.